api.model.js

```diff
@@ -0,0 +1,11 @@
+const db = require('./db/connection')
+
+exports.topicFinder = () => {
+    const query = 'SELECT * FROM topics;';
+
+    return db.query(query).then((result)=>{
+        console.log(result.rows)
+        return result.rows;
```

Comment:
when receiving an object as a response we can destructure the object to save the value of the key onto a variable of the same name.